my/time/tz/via_location.py

```diff
@@ -71,12 +98,13 @@ def _iter_local_dates(start=0, stop=None) -> Iterator[DayWithZone]:
         yield DayWithZone(day=ndate, zone=z)
 
 
-def most_common(l):
-    res, count = Counter(l).most_common(1)[0] # type: ignore[var-annotated]
+def most_common(lst: List[DayWithZone]) -> DayWithZone:
+    res, _ = Counter(lst).most_common(1)[0]  # type: ignore[var-annotated]
     return res
 
 
-@mcachew(cache_path=cache_dir())
+# refresh _iter_tzs once per day -- don't think a better depends_on is possible dynamically
+@mcachew(logger=logger, depends_on=lambda: str(date.today()))
```

Comment:
Not sure about this, but otherwise it just never updates as far as I understand

Reply:
ouch. yeah interesting, something must have been lost during refactoring -- I might have not used this for a while